src/docsfy/api/auth.py

```
@@ -0,0 +1,168 @@
+        body = await request.json()
+    except Exception as exc:
+        raise HTTPException(status_code=400, detail="Invalid JSON body") from exc
+    if not isinstance(body, dict):
+        raise HTTPException(
+            status_code=400, detail="Request body must be a JSON object"
+        )
+    username = str(body.get("username", ""))
+    api_key = str(body.get("api_key", ""))
+    settings = get_settings()
+
+    logger.debug(f"Login attempt for username '{username}'")
```

Ruff (0.15.6)

[warning] Logging statement uses f-string

(G004)